discord/ext/converters/range.py

```diff
@@ -0,0 +1,13 @@
+from typing import Tuple
+
+from discord.ext import commands
+
+
+class InRangeConverter(commands.Converter):
+    def __init__(self, *, range_tuple: Tuple) -> None:
+        self.range = range(*range_tuple)
+
+    async def convert(self, ctx, argument: str) -> str:
+        if argument in self.range:
```

Comment:
A `str` will never exist in a `range`.

```suggestion
        if argument in string.digits and int(argument) in self.range:
```

Reply:
:man_facepalming: I don't know how I made such a rookie mistake. :fearful: